packages/inquirer-gui/__tests__/confirm.spec.js

```diff
@@ -11,6 +11,8 @@ const questionConfirm = [
     type: "confirm",
     name: "confirm",
     message: "Are you sure?",
+    labelTrue: "Accept",
+    labelFalse: "Decline",
```

Comment:
what is the purpose of this change? I don't see this test checking for overriding the label value.